workflows/utils.py

```
@@ -1,15 +1,14 @@
 import json
 import logging
-from logging import Logger
-from settings import settings
-from typing import Tuple, Any
+import logging
```

Ruff (0.8.2)

Redefinition of unused `logging` from line 2

Remove definition: `logging`

(F811)